faq.md

```diff
@@ -0,0 +1,46 @@
+### What is the Jetpack Start API? ###
```

Comment:
I haven't previously seen heading with the `###` before and after the heading. Normally, I'd expect this to be written as:

```
### What is the Jetpack Start API? ###
```